### PR TITLE
Release 1.17.0

### PR DESCRIPTION
**Pull request by jeffstieler**
https://github.com/Automattic/woocommerce-services/compare/v1.16.1...master

### CODE REVIEW

**Comment by v18** [comment]
This plugin review is on hold:
- to get all of the Calypso changes, courtesy of @DanReyLop 
- to fix the issue where the packages list is unavailable

**Comment by jeffstieler** [comment]
Calypso changes have been merged into master.

**Comment by v18** [comment]
Packages list is being fixed in the server - PR open.

**Comment by v18** [comment]
> Calypso changes have been merged into master.

What's the PR for this?

**Comment by DanReyLop** [comment]
I've just rebased this PR with `master`. Wait before merging, there are a few more things to include in the changelog.

File: readme.txt [hunk 1]
```diff
@@ -82,6 +82,13 @@ As of the WooCommerce 3.5 release, WooCommerce Services no longer provides shipp
 
 == Changelog ==
 
+= 1.17.0 =
+* Restrict shipping methods to certain shipping classes
+* Improve Stripe onboarding and show connected account in settings
+* Fix label printing bug in iOS 12
+* Make the HS Tarriff Code optional for international shipments
```
Comment (dechov) [inline readme.txt]:
```suggestion
* Make the HS Tariff Code optional for international shipments
```

**Pull request merged**